### PR TITLE
added styles for dropdowns and dependencies

### PR DESCRIPTION
**Pull request by PaulSchroeder**
this pull request includes commits to setup documentation page and basic typography styles

### CODE REVIEW

**Review by danieleguido — CHANGES_REQUESTED** [review]
Had to fix a few vulnerabilities and two eslint errors?
Should work now, but please double check :)

File: src/App.vue [hunk 1]
```diff
@@ -20,11 +22,6 @@ export default {
 
 <style>
 #app {
-  font-family: 'Avenir', Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-  margin-top: 60px;
+  margin: 1em;
```
Comment (tvanbeek) [inline src/App.vue:25]: better use the bootstrap utility classes like on the impresso-frontend

Comment (tvanbeek) [reply src/App.vue:25]: `<div id="app" class="m-1">`

File: src/scss/bootpresso.scss [hunk 2]
```diff
@@ -1,7 +1,9 @@
 // customizing
-@import "variables.scss";
+@import "variables.sass";
+@import "typography.sass";
+@import "vendor/dripicons";
```
Comment (tvanbeek) [inline src/scss/bootpresso.scss:4]: vendor/dripicons should not be in the folder src/scss but in a separate src/vendor folder, and than maybe refer to it like so `@import "@/src/vendor/dripicons";`

File: src/scss/bootpresso.scss [hunk 3]
```diff
@@ -37,3 +39,7 @@
 @import "bootstrap/scss/_carousel.scss";
 @import "bootstrap/scss/_utilities.scss";
 @import "bootstrap/scss/_print.scss";
+
+// custom components
```
Comment (tvanbeek) [inline src/scss/bootpresso.scss:43]: what does custom refer to here? is it still a regular bootstrap component with custom styling? or a custom component?

**Review by tvanbeek — CHANGES_REQUESTED** [review]

**Pull request merged**